src/Library/demos/Stacks/main.blp

```diff
@@ -0,0 +1,108 @@
+using Gtk 4.0;
+using Adw 1;
+
+Box root_box {
+  orientation: vertical;
+  halign: center;
+  valign: center;
+  Stack stack {
+
+    StackPage page1 {
+      title: _("Page 1");
+      child: Adw.StatusPage {
+        title: _("Stacks");
+        description: _("only shows one of its children at a time");
+
+        Label {
+          label: _("Documentation");
+          margin-top: 24;
+          styles ["title-2"]
+        }
+
+        Box {
+          orientation: horizontal;
+          halign: center;
+          margin-top: 24;
+
+          LinkButton {
+            label: _("Stack");
+            uri: "https://docs.gtk.org/gtk4/class.Stack.html";
+          }
+
+          LinkButton {
+            label: _("Stack Sidebar");
+            uri: "https://docs.gtk.org/gtk4/class.StackSidebar.html";
+          }
+
+          LinkButton {
+            label: _("Stack Switcher");
+            uri: "https://docs.gtk.org/gtk4/class.StackSwitcher.html";
+          }
+        }
+      };
+    }
+
+    StackPage page2 {
+      title: _("Page 2");
+      child: Adw.StatusPage {
+        Adw.PreferencesGroup {
+          styles ["boxed-list"]
+          halign: center;
```

Comment:
```suggestion
          halign: center;
```

I think this should be left to default (`fill`), or given a minimum width, because the rows get a little cramped depending on the dropdown selection.